components/zoho_bugtracker/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "@pipedream/zoho_bugtracker",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "description": "Pipedream Zoho BugTracker Components",
-  "main": "dist/app/zoho_bugtracker.app.mjs",
+  "main": "zoho_bugtracker.app.mjs",
   "keywords": [
     "pipedream",
     "zoho_bugtracker"
   ],
-  "files": ["dist"],
   "homepage": "https://pipedream.com/apps/zoho_bugtracker",
   "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
   "publishConfig": {
     "access": "public"
-  }
-}
\ No newline at end of file
+  },
+  "dependencies": {}
```

Comment:
There are no dependencies listed, but at least `pipedream/platform` and `form-data` are used, is this ok?